src/github/client/github-installation-client.ts

```diff
@@ -150,7 +147,7 @@ export class GitHubInstallationClient extends GitHubClient {
 	 */
 	public getRepositoriesPage = async (per_page = 1, cursor?: string): Promise<GetRepositoriesResponse> => {
 		try {
-			const response = await this.graphql<GetRepositoriesResponse>(GetRepositoriesQuery, {
+			const response = await this.graphql<GetRepositoriesResponse>(GetRepositoriesQuery, await this.installationAuthenticationHeaders(), {
```

Comment:
I'd think making `appAuthenticationHeaders()` or `getAxiosConfig()` as abstract in the parent class, (but impl it in child class) will be cleaner, so that in the actual call of `this.graphql`, we don't need to remember to pass in the config.

Reply:
Yeah, that would be nice, but I think it is better to do that once we have more methods that will use it. Currently, that's only one and I prefer to be explicit here, therefore calling this YAGNI for now 😋